bin/dsmd.pl

```diff
@@ -547,14 +547,19 @@ sub run {
     );
     $self->{db_centstorage}->connect();
 
-    $self->load_slot_locks();
+    my $moduleDsm = $self->{db_centstorage}->query("select count(name) from centreon.modules_informations where name = 'centreon-dsm'");
```

Comment:
You can't hardcoded `centreon` database
Some users will use databases with other names

Reply:
variable created

Reply:
variable deleted, used configuration instead